virttest/utils_libvirt/libvirt_monitor.py

```diff
@@ -0,0 +1,39 @@
+import copy
+import logging
+
+from avocado.core import exceptions
+
+from virttest import virsh
+
+LOG = logging.getLogger('avocado.' + __name__)
+
+
+def check_domjobinfo(params, expected_list, remote=False):
+    """
+    Check domjobinfo
+
+    :param params: dict, get vm name
+    :param expected_list: expected list
```

Comment:
Could you make the docstring more informative?

Reply:
Updated, thanks.